docker/image/git_add_files.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import argparse
+import git
+import re
+
+parser = argparse.ArgumentParser(
+    description='Add selected files to git repository.')
+parser.add_argument('repository_path', action='store')
+args = parser.parse_args()
+repo = git.Repo(args.repository_path)
+
+git_diff_numstat = repo.git.diff(numstat=True)
+git_diff = repo.git.diff().encode("utf-8")
+
+# Parse information from git's numstat
+modified_files = {}
+for line in git_diff_numstat.splitlines():
+    tmp = re.split(r'\t+', line)
```

Comment:
@apojomovsky consider using de-structuring assignment for better readability:

```python
field_1, field_2, field_3 = re.split(pattern, string)[:3]
```

Reply:
that sounds great, thanks for suggesting it! done!